README.md

```diff
@@ -98,6 +98,15 @@ KlaviyoClient.profiles.getProfileMetricsTimelineById({
     count: 10,
     sort: 'desc'
 });
+
+//Unset properties on a profile.
```

Comment:
nit: think you want a space between the slashes and the first word of the comment.